bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt

```diff
@@ -197,7 +197,10 @@ class ModelImporter(
             // optimization for when there is no change in the child list
             // size check first to avoid querying the original ID
             if (expectedNodes.size == existingNodes.size && expectedNodes.map { it.originalId() } == existingNodes.map { it.originalId() }) {
-                existingNodes.zip(expectedNodes).forEach { syncNode(it.first, it.second, progressReporter) }
+                existingNodes.zip(expectedNodes).forEach {
+                    checkAndHandleConceptChange(it.first, it.second)
+                    syncNode(it.first, it.second, progressReporter)
```

Comment:
This pair of `checkAndHandleConceptChange` occurs here and in line 258. Could probably be combined in a method or the concept change could be made part of `syncNode`?

Reply:
It's part of `syncNode` now.